BUILDING.md

```diff
@@ -233,24 +233,25 @@ The Node.js project supports Python >= 3 for building and testing.
 
 #### Unix prerequisites
 
-* `gcc` and `g++` >= 8.3 or newer, or
+* `gcc` and `g++` >= 8.3 or newer
 * GNU Make 3.81 or newer
 * Python 3.6, 3.7, 3.8, 3.9, or 3.10 (see note above)
+  * For test coverage, your Python installation must include pip.
 
 Installation via Linux package manager can be achieved with:
 
-* Ubuntu, Debian: `sudo apt-get install python3 g++ make`
-* Fedora: `sudo dnf install python3 gcc-c++ make`
-* CentOS and RHEL: `sudo yum install python3 gcc-c++ make`
-* OpenSUSE: `sudo zypper install python3 gcc-c++ make`
-* Arch Linux, Manjaro: `sudo pacman -S python gcc make`
+* Ubuntu, Debian: `sudo apt-get install python3 python3-pip g++ make`
+* Fedora: `sudo dnf install python3 python3-pip gcc-c++ make`
+* CentOS and RHEL: `sudo yum install python3 python3-pip gcc-c++ make`
+* OpenSUSE: `sudo zypper install python3 python3-pip gcc-c++ make`
+* Arch Linux, Manjaro: `sudo pacman -S python python-pip gcc make`
 
 FreeBSD and OpenBSD users may also need to install `libexecinfo`.
 
 #### macOS prerequisites
 
 * Xcode Command Line Tools >= 11 for macOS
-* Python 3.6, 3.7, 3.8, 3.9, or 3.10 (see note above)
+* Python 3.6, 3.7, 3.8, 3.9, or 3.10 (see note above) with pip installed
```

Comment:
This and the package references above should be changed as well.

Reply:
I've updated this and moved the pip install packages to the end of package list (so it's slightly easier to remove them if someone doesn't want them). 

It feels that it's better to keep them together than make the install instructions longer with 2 versions, but please let me know if you disagree.